src/web/index.ts

```diff
@@ -69,14 +69,17 @@ export default class PubNub extends PubNubCore<ArrayBuffer | string, PubNubFileP
       clientConfiguration.keepAlive,
       clientConfiguration.logVerbosity!,
     );
-    if (configurationCopy.serviceWorkerUrl) {
-      // Inject subscription service worker into transport provider stack.
-      transport = new SubscriptionServiceWorkerMiddleware({
+
+    if (configurationCopy.subscriptionWorkerUrl) {
+      // Inject subscription worker into transport provider stack.
+      transport = new SubscriptionWorkerMiddleware({
         clientIdentifier: clientConfiguration._instanceId,
         subscriptionKey: clientConfiguration.subscribeKey,
-        serviceWorkerUrl: configurationCopy.serviceWorkerUrl,
+        userId: clientConfiguration.getUserId(),
+        workerUrl: configurationCopy.subscriptionWorkerUrl,
         sdkVersion: clientConfiguration.getVersion(),
         logVerbosity: clientConfiguration.logVerbosity!,
+        workerLogVerbosity: platformConfiguration.subscriptionWorkerLogVerbosity!,
```

Comment:
I'm assuming that `platformConfiguration.subscriptionWorkerLogVerbosity` will always hold non-null value right?

Reply:
@mohitpubnub yup, there are defaults applied when final platform-specific configuration created [here](https://github.com/pubnub/javascript/blob/feat/use-shared-worker/src/web/components/configuration.ts#L120).